File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/metadata/read/MetadataReadRequestTest.java
package uk.ac.jisc.rdss.tests.metadata.read;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class MetadataReadRequestTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/metadata/read/request_schema.json";
    }

    @Test
    public void validateRequest() throws Exception {
        validateJson("messages/metadata/read/request.json");
    }
}

File: message-api-schema-validator/tests/metadata/metadata_read_request_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class MetadataReadRequestTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return 'messages/body/metadata/read/request_schema.json'

    def runTest(self):
        self.validate_json('messages/body/metadata/read/request.json')
